docs/devices/YSR-MINI-01.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "YSRSAI ZB-CL01 control via MQTT"
+description: "Integrate your YSRSAI ZB-CL01 via Zigbee2mqtt with whatever smart home
+ infrastructure you are using without the vendors bridge or gateway."
+---
+
+*To contribute to this page, edit the following
+[file](https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/devices/ZB-CL01.md)*
```

Comment:
You changed file name, but this link is with old name, so it will be broken.

Reply:
No problem, this file we be regenerated automatically on the next zigbee2mqtt release (I will add the pairing section to https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docgen/device_page_notes.js)